libraries/AP_Logger/LogFile.cpp

```diff
@@ -244,6 +244,41 @@ void AP_Logger::Write_RCOUT(void)
         chan14        : hal.rcout->read(13)
     };
     WriteBlock(&pkt, sizeof(pkt));
+
+#if NUM_SERVO_CHANNELS >= 15
+    const struct log_RCOUT2 pkt2{
+        LOG_PACKET_HEADER_INIT(LOG_RCOUT2_MSG),
+        time_us       : AP_HAL::micros64(),
+        chan15         : hal.rcout->read(14),
+        chan16         : hal.rcout->read(15),
+        chan17         : hal.rcout->read(16),
+        chan18         : hal.rcout->read(17),
+    };
+    WriteBlock(&pkt2, sizeof(pkt2));
+#endif
+
+#if NUM_SERVO_CHANNELS >= 19
+    const struct log_RCOUT pkt3{
+        LOG_PACKET_HEADER_INIT(LOG_RCOUT2_MSG),
```

Comment:
```suggestion
        LOG_PACKET_HEADER_INIT(LOG_RCOUT19_32_MSG),
```

I'd also rather see these all labeled accurately chan19, chan20, chan21... instead of reusing index chan1,chan2,chan3.. It will just lead to confusion

Reply:
Note, I believe the author's intent was to use LOG_RCOUT3_MSG

Reply:
Yep, I meant to use `LOG_RCOUT3_MSG`. I figured it was not worth having the duplicate log structure just to have the channel names match.